tests/migration_test.go

```diff
@@ -122,7 +122,7 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 			return err
 		}, timeout, 1*time.Second).ShouldNot(HaveOccurred())
 		By("Waiting until the VirtualMachineInstance starts")
-		tests.WaitForSuccessfulVMIStartWithTimeout(obj, timeout)
+		tests.WaitForSuccessfulVMIStartWithTimeoutIgnoreWarnings(obj, timeout)
```

Comment:
what are the warnings that pop up here? do we want to make this "ignore warnings" a boolean so it doesn't impact other tests?